src/MongoDB.Analyzer/Core/Utilities/AnalysisCodeGeneratorUtilities.cs

```diff
@@ -13,20 +13,23 @@
 // limitations under the License.
 
 using MongoDB.Analyzer.Core.HelperResources;
+using NuGet.Versioning;
 
 namespace MongoDB.Analyzer.Core.Utilities;
 
 internal static class AnalysisCodeGeneratorUtilities
 {
-    private const string AnalysisAssemblyName = "DynamicProxyGenAssembly2";
+    private const string AnalysisAssemblyNameDRIVER_2_27_OR_Lower = "DynamicProxyGenAssembly2";
+    private const string AnalysisAssemblyNameDRIVER_2_28_OR_Greater = "MongoDB.Analyzer.MQLGenerator";
+    private const string Driver_V2_28_OrGreater = "[2.28.0, )";
```

Comment:
Let's have consistent naming style:
 28_OR_GREATER for ifdef directives and configurations. And OrLower/OrGreater for code variables.

Reply:
Done